modules/context/api.go

```diff
@@ -74,8 +75,8 @@ func (ctx *APIContext) Error(status int, title string, obj interface{}) {
 		message = obj.(string)
```

Comment:
Just thinking on this might be nicer

```suggestion
		message = fmt.Sprintf("%s", obj)
```